instance/models/openedx_instance.py

```diff
@@ -281,6 +282,19 @@ def first_activated(self):
         except models.ObjectDoesNotExist:
             return None
 
+    @property
+    def most_recently_archived(self):
```

Comment:
I think the naming of this property is not the most fortunate. How about renaming it to something like `latest_archiving_date`?

```suggestion
    def latest_archiving_date(self):
```

if you agree, please make sure to update the management command as well.

Reply:
I personally don't love it when names are unspecific with regards to types (i.e. "date" vs "datetime") but I am following your suggestion because you know the teams naming conventions.